src/suggestion.rs

```diff
@@ -394,8 +677,150 @@ mod tests {
+                    }
+                }
+            },
+        );
+
+        let suggestion = Suggestion {
+            detector: Detector::Dummy,
+            origin: ContentOrigin::TestEntity,
+            chunk: &chunk,
+            range: 6..12,
+            span: Span {
+                start: LineColumn {
+                    line: 1,
+                    column: 10,
+                },
+                end: LineColumn {
+                    line: 1,
+                    column: 15,
+                },
+            },
+            replacements: vec!["replacement_0", "replacement_1", "replacement_2"]
+                .into_iter()
+                .map(std::borrow::ToOwned::to_owned)
+                .collect(),
+            description: Some("Possible spelling mistake found.".to_owned()),
+        };
+
+        const EXPECTED: &'static str = r#"error: spellcheck(Dummy)
+  --> /tmp/test/entity:1
+   |
+ 1 |  Line mitake 1
+   |       ^^^^^^
+   | - replacement_0, replacement_1, or replacement_2
+   |
+   |   Possible spelling mistake found.
+   |
+"#;
+
+        assert_display_eq(suggestion, EXPECTED);
+    }
+
+    #[test]
+    fn fmt_2_multi_80_plus() {
+        const CONTENT: &'static str = r#" Line mitake 1
+ Suuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuper duuuuuuuuuuuuuuuuuuuuuuuuper too long
+ "#;
+
+        let chunk = CheckableChunk::from_str(
+            CONTENT,
+            indexmap::indexmap! {
+                0..13 => Span {
+                    start: LineColumn {
+                        line: 1,
+                        column: 4,
+                    },
+                    end: LineColumn {
+                        line: 1,
+                        column: 16,
+                    }
+                },
+                14..101 => Span {
+                    start: LineColumn {
+                        line: 2,
+                        column: 4,
+                    },
+                    end: LineColumn {
+                        line: 2,
+                        column: 90,
+                    }
+                }
+            },
+        );
+
+        let suggestion = Suggestion {
+            detector: Detector::Dummy,
+            origin: ContentOrigin::TestEntity,
+            chunk: &chunk,
+            range: 66..94,
+            span: Span {
+                start: LineColumn { line: 2, column: 5 },
+                end: LineColumn {
+                    line: 2,
+                    column: 92,
+                },
+            },
+            replacements: vec!["replacement_0", "replacement_1", "replacement_2"]
+                .into_iter()
+                .map(std::borrow::ToOwned::to_owned)
+                .collect(),
+            description: Some("Possible spelling mistake found.".to_owned()),
+        };
+
+        const EXPECTED: &'static str = r#"error: spellcheck(Dummy)
+  --> /tmp/test/entity:2
+   |
+ 2 | ..uuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuper duuu...uper too long
```

Comment:
two spaces are being displayed in previous test and one space is given till the sentence starts here.

Reply:
Slightly different, the literals above contain a leading space (the one `/// <this one`) and thus it appears as two, but it really is just one plus the literal lead by a space, in case of condensation we start with `..` and hence only one <kbd>space</kbd> before the line.

Reply:
when this is calculated `padding_till_excerpt_start` it is considering two extra spaces in the beginning because I thought that the sentence should start from two white spaces, so maybe needs to be adjusted.

Reply:
but not sure as it seems to work accordingly just a note about it.